gravitee-apim-gateway/gravitee-apim-gateway-policy/src/test/java/io/gravitee/gateway/policy/impl/PolicyLoaderTest.java

```diff
@@ -0,0 +1,111 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.policy.impl;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.gravitee.definition.model.Policy;
+import io.gravitee.gateway.core.classloader.DefaultClassLoader;
+import io.gravitee.gateway.core.component.CustomComponentProvider;
+import io.gravitee.gateway.policy.PolicyManifest;
+import io.gravitee.gateway.policy.dummy.*;
+import io.gravitee.plugin.core.api.ConfigurablePluginManager;
+import io.gravitee.plugin.core.api.PluginManager;
+import io.gravitee.plugin.core.api.PluginManifest;
+import io.gravitee.plugin.policy.PolicyPlugin;
+import io.gravitee.plugin.policy.internal.PolicyClassLoaderFactoryImpl;
+import io.gravitee.plugin.policy.internal.PolicyPluginManagerImpl;
+import io.gravitee.policy.api.PolicyContext;
+import io.gravitee.policy.api.annotations.OnRequest;
+import io.gravitee.policy.api.annotations.OnRequestContent;
+import io.gravitee.policy.api.annotations.OnResponse;
+import io.gravitee.policy.api.annotations.OnResponseContent;
+import java.net.URL;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author Guillaume LAMIRAND (guillaume.lamirand at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+class PolicyLoaderTest {
+
+    private PolicyLoader policyLoader;
+
+    @BeforeEach
+    public void init() {
+        ConfigurablePluginManager policyPluginManager = mock(ConfigurablePluginManager.class);
+        when(policyPluginManager.get("dummy-policy"))
+            .thenReturn(new DummyPolicyPlugin("dummy-policy", DummyPolicy.class, DummyPolicyConfiguration.class, DummyPolicyContext.class));
+        when(policyPluginManager.get("dummy-reactive"))
+            .thenReturn(
+                new DummyPolicyPlugin("dummy-reactive", DummyReactivePolicy.class, DummyPolicyConfiguration.class, DummyPolicyContext.class)
+            );
+        policyLoader =
+            new PolicyLoader(
+                new DefaultClassLoader(this.getClass().getClassLoader()),
+                policyPluginManager,
+                new PolicyClassLoaderFactoryImpl(),
+                new CustomComponentProvider()
+            );
+    }
+
+    @Test
+    public void should_return_empty_loaded_policies_from_empty_dependencies() {
```

Comment:
To keep consistency across all projects, can we keep the camel case practice ?

Reply:
that fact is there is no consistency across projects and it is hard to understand what a test actually test.